mantidproject/mantid
# Update algorithms for Python 2/3 compatibility, group 2

Group 2 of #16739.

**To test:**
- check code changes
- tests should still pass (Python 2)
- If someone has a Python 3 Mantid setup, the tests should also pass there

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
